awx/main/credential_plugins/aim.py

```diff
@@ -74,6 +80,7 @@ def aim_backend(**kwargs):
     app_id = kwargs['app_id']
     object_query = kwargs['object_query']
     object_query_format = kwargs['object_query_format']
+    object_property = kwargs['object_property']
```

Comment:
@infamousjoeg will that work if I do not provide the `object_property` key in metadata: 

```
{
  "object_query": "Safe=TestSafe;Object=testAccountName123",
  "object_query_format": "Exact",
}
```
I'm just thinking about backward compatibility with the possible existing credentials without `object_property`?

Reply:
Yes, it defaults to using the "Content" key value instead which is how it works today.  So, a change will not break current integrations.

Reply:
I think this comment is still actual. I think we need to keep compatibility with old API calls. So when we call API and we do not provide object_property in the inputs we gonna fail here saying that the object_property is not in the kwargs. 

I think we should be more careful here and use the default value: 
```
object_property = kwargs.get('object_property', '')
```

Reply:
I have tested this successfully and committed the change.

Reply:
Tested and verified as fixed.